src/config/constants/campaigns.ts

```diff
@@ -0,0 +1,55 @@
+import { Campaign } from './types'
+
+/**
+ * id: The campaign id (required)
+ * type: The type of the achievement
+ * title: A string or an object to be translated.
+ * Note: If string it could be used as a variable in a translation e.g. IFOs
+ *
+ * badge: Achievement avatar
+ */
+
+const campaigns: Campaign[] = [
+  {
+    id: '511050000',
+    type: 'ifo',
+    title: 'Soteria',
+    badge: 'ifo-wsote.svg',
+  },
```

Comment:
The idea here is keep campaign meta separate because they are eventually going to be many different types.